templates/display_new.tt.html

```diff
@@ -0,0 +1,272 @@
+<!doctype html>
+<html class="no-js" lang="[% language %]" data-serverdomain="[% server_domain %]">
+<head>
+    <meta charset="utf-8">
+    <title>[% title %]</title>
+    [% meta_description %]
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta property="fb:app_id" content="219331381518041">
```

Comment:
Could you move this to a config setting, like the for example the `twitter_account` value?